data/presets/presets/amenity/shelter/public_transport.json

```diff
@@ -0,0 +1,28 @@
+{
+    "icon": "maki-shelter",
+    "fields": [
+        "name",
+        "shelter_type",
+        "building_area",
+        "bench",
+        "lit"
+    ],
+    "geometry": [
+        "point",
+        "area"
+    ],
+    "terms": [
+        "bus stop",
+        "metro stop",
+        "waiting"
```

Comment:
A bus stop is something different than a shelter and tagged differently (a preset exists already). I think that this will let users use the wrong preset to add bus stops.